doc/ConsoleCtrlEvent.md

```diff
@@ -2,7 +2,7 @@
 
 ## Generation
 
-conhost requests that user32 inject a thread into the attached console application.
+conhost requests that user32 injects a thread into the attached console application.

```

Comment:
Somebody wanted to change this in the past and it was said that it is correct english as is. Since I am no native speaker I am unsure what is correct.

Reply:
Yes, “requests that user32 inject” is correct English. “Injects” is slightly less correct.